### ISSUE TITLE
Problems in training my dataset

### ISSUE THREAD

**Issue by wenze-shui** [bonlime/keras-deeplab-v3-plus#129]
When training my own data set, I encountered problems in the graph. After the first epoch, the loss became very low, but the accuracy has been stopped at loss than 30%.
![image](https://user-images.githubusercontent.com/44524397/70236970-fd4b5400-17a0-11ea-9b37-3a3849596cfe.png)

**Comment by wenze-shui** [comment]
This is the output of training
Epoch 1/100
3750/3750 [==============================] - 1993s 532ms/step - loss: 0.5501 - acc: 0.3078 - val_loss: 1.1921e-07 - val_acc: 0.2953
Epoch 2/100
3750/3750 [==============================] - 1961s 523ms/step - loss: 1.1921e-07 - acc: 0.2943 - val_loss: 1.1921e-07 - val_acc: 0.2953
Epoch 3/100
3750/3750 [==============================] - 1962s 523ms/step - loss: 1.1921e-07 - acc: 0.2943 - val_loss: 1.1921e-07 - val_acc: 0.2953
Epoch 4/100
3750/3750 [==============================] - 1972s 526ms/step - loss: 1.1921e-07 - acc: 0.2943 - val_loss: 1.1921e-07 - val_acc: 0.2953

**Comment by ottovonzastrow** [comment]
Same problem for me - potentially a vanishing/ exploding gradient problem?

**Comment by TWJubb** [comment]
You need to specify the activation function of the last layer when you create the model; it defaults to `None` which will mean you get unbounded numbers out which will throw a loss function like `categorical_crossentropy`.

For me this led to the same problems as you guys have above. I used `activation="softmax"`  in the call to `DeeplabV3` `to fix it.
